addons/wowl/static/src/components/dropdown/dropdown_item.ts

```diff
@@ -0,0 +1,49 @@
+import { Component } from "@odoo/owl";
+
+export enum ParentClosingMode {
+  None = "none",
+  ClosestParent = "closest",
+  AllParents = "all",
+}
+
+export interface DropdownItemClickedPayload {
+  payload: Object;
+  dropdownClosingRequest: {
+    mode: ParentClosingMode;
+    isFresh: boolean;
+  };
+}
+
+export class DropdownItem extends Component {
+  static template = "wowl.DropdownItem";
+  static props = {
+    onClickPayload: {
+      type: Object,
+      optional: true,
+    },
+    parentClosingMode: {
+      type: ParentClosingMode,
+      optional: true,
+    },
+  };
+  static defaultProps = {
+    parentClosingMode: ParentClosingMode.AllParents,
+  };
+
+  /**
+   * Handlers
+   */
+  fire(ev: MouseEvent) {
```

Comment:
This should just be an "onClick" or "onSelected". So that you know what it is without checking the xml. After all, this is a click handler, not a method you call somewhere.

Reply:
True.